assets/src/edit-story/elements/media/editPanMoveable.js

```diff
@@ -47,21 +50,58 @@ function EditPanMoveable({
   const translateRef = useRef([0, 0]);
   const transformFlip = getTransformFlip(flip);
 
-  const update = () => {
+  const update = useCallback(() => {
     const [tx, ty] = translateRef.current;
     fullMedia.style.transform = `translate(${tx}px, ${ty}px) ${
       transformFlip ?? ''
     }`;
     croppedMedia.style.transform = `translate(${tx}px, ${ty}px) ${
       transformFlip ?? ''
     }`;
-  };
+  }, [croppedMedia, fullMedia, transformFlip]);
 
   // Refresh moveables to ensure that the selection rect is always correct.
   useEffect(() => {
     moveableRef.current.updateRect();
   });
 
+  useKeyDownEffect(
+    { current: croppedMedia },
```

Comment:
This isn't pretty. But I'm not sure of the consequences if any.

@dvoytenko, any comments about this fake inline reference?

Reply:
`useKeyDownEffect` should accept either ref or a node so it should be ok to just pass `croppedMedia` directly as well.

Reply:
Ah, much better - thanks! 🎉 🙏 

Fixed in 5b7031ec4.